src/components/input/color/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Element, Event, EventEmitter, h, Host, Method, Prop, VNode } from "@stencil/core"
+import { Color } from "../../../model"
+import { Clearable } from "../Clearable"
+import { Input } from "../Input"
+import { Looks } from "../Looks"
+
+@Component({
+	tag: "smoothly-input-color",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyInputColor implements Input, Clearable {
+	@Element() element: HTMLElement
+	@Prop({ mutable: true, reflect: true }) value: string | undefined
+	@Prop({ mutable: true, reflect: true }) looks: Looks = "plain"
+	@Prop() name: string
+	@Event() smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color) => void>
+	@Event() smoothlyInput: EventEmitter<Record<string, any>>
```

Comment:
I don't think we are emitting  this anywhere? add watcher on value to emit.
Do we know if we want to emit the current value in `componentWillLoad` to send a message to other components like the form. or have this not yet been discussed?

Reply:
not yet discussed but ill fix that now!